patches/tModLoader/Terraria/Player.cs.patch

```diff
@@ -1608,15 +1614,21 @@
 +			for (int k = 0; k < 10; k++) {
 +				if (IsAValidEquipmentSlotForIteration(k))
 +					VanillaUpdateEquip(armor[k]);
- 			}
++			}
++
++			LoaderManager.Get<AccessorySlotLoader>().VanillaUpdateEquipsMirror(this);
```

Comment:
I'd prefer if all equip related `AccessorySlotLoader` functionality was implemented via `ModAccessorySlotPlayer `. That way we can ensure feature parity with mod accessory slots, and the `ModPlayer` api in terms of having equipment effects/visuals applied from custom sources.